ur_bringup/launch/ur_control.launch.py

```diff
@@ -225,6 +225,19 @@ def launch_setup(context, *args, **kwargs):
         ],
     )
 
+    controller_stopper_node = Node(
+        package="controller_stopper",
+        executable="controller_stopper_node",
+        name="controller_stopper",
+        output="screen",
+        emulate_tty=True,
+        parameters=[
+            {"headless_mode": headless_mode},
+            {"joint_controller_active": activate_joint_controller},
+            {"consistent_controllers": ["io_and_status_controller"]},
```

Comment:
This list should be extended if you agree with my comment on the "broadcaster" string matching.